docs/book/src/reference/watching-resources.md

```diff
@@ -0,0 +1,10 @@
+# Watching Resources
+
+Inside a `Reconcile()` control loop, controllers will often use the state of the kubernetes cluster, and resources in it, to make decisions.
+Therefore, it can be necessary to know when a resource that you care about is changed.
```

Comment:
```suggestion
Inside a `Reconcile()` control loop, you are looking to do a collection of operations until it has the desired state on the cluster. Therefore, it can be necessary to know when a resource that you care about is changed.
```